lib/scrabble.rb

```diff
@@ -1,7 +1,21 @@
+require_relative 'word_finder'
+
+# scrabble scoring class
 class Scrabble
+  def initialize
+    @finder = WordFinder.new
+  end
 
   def score(word)
-    1
+    if !word || !@finder.exists?(word)
```

Comment:
Avoid using not in conditionals. This can also be better represented with a predicate method.